index.html

```diff
@@ -1,81 +1,55 @@
 <!DOCTYPE html>
 
-<html>
+<html lang="en">
 <head>
-  <script type="module" src="dist/byu-theme-components.min.js"></script>
-  <link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/ringside/fonts.css" media="all">
-  <link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/public-sans/fonts.css" media="all">
-  <style>
-    html, body {
-      height: 100%;
-      margin: 0;
-      padding: 0;
-      font-family: "Public Sans", Arial, sans-serif;
-    }
+    <script type="module" src="dist/byu-theme-components.min.js"></script>
+    <link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/ringside/fonts.css" media="all">
+    <link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/public-sans/fonts.css" media="all">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        html, body {
+            height: 100%;
+            margin: 0;
+            padding: 0;
+        }
 
-    .containing-element {
-      display: flex;
-      flex-direction: column;
-      height: 100%;
-    }
+        .containing-element {
+            display: flex;
+            flex-direction: column;
+            height: 100%;
+        }
 
-    .page-content {
-      flex-grow: 1;
-    }
-  </style>
-  <title>BYU Theme Components</title>
+        .page-content {
+            flex-grow: 1;
+        }
+    </style>
+    <title>BYU Theme Components</title>
 </head>
 <body>
 <div class="containing-element">
-  <byu-header></byu-header>
-  <div class="page-content">
+    <byu-header>
+        <a href="//humanities.byu.edu" slot="breadcrumb">Humanities</a>
+        <span slot="site-title">Demo Site</span>
+        <!-- TODO: Implement subititle class -->
+        <span slot="site-subtitle">This is a very cool demo site.</span>
+        <a href="//donate.byu.edu" slot="actions">Donate</a>
+        <byu-search slot="search"></byu-search>
+        <byu-user-info slot="user">
+            <a slot="user-name" href="https://my.byu.edu">pparker</a>
+            <a slot="logout" href="http://cas.byu.edu/cas/logout">Sign out</a>
+        </byu-user-info>
+        <byu-menu slot="nav">
+            <a href="/">Home</a>
+            <a href="/about">About</a>
+            <a href="/programs">Majors &amp; Programs</a>
+            <a href="/students">Students</a>
+            <a href="/alumni">Alumni</a>
+            <a href="/giving">Giving</a>
+        </byu-menu>
+    </byu-header>
+    <div class="page-content">
         <p>This is the page.</p>
-  </div>
-  <byu-footer>
```

Comment:
I know this PR focuses on the header, but is there a reason the footer example was removed here?

Reply:
Because we need to redo the footer. It hasn't been done in fractal yet, so I originally ended up just copying the mess from the original components into the new version which is probably not what we should do.

Reply:
I think it'd be better to say we need to cleanup the SCSS for the footer (which is definitely true) - as I understand it the display requirements haven't changed, so when we're done it should look the same as the old footer, just use better structured code in the background.

Reply:
Moving this conversation to Slack.